magma/conversions.py

```diff
@@ -191,6 +191,8 @@ def concat(*arrays):
     for a in arrays:
         if isinstance(a, hwtypes.BitVector):
             ts.extend(a.bits())
+        elif isinstance(a, Bit):
+            ts.extend(bits(a).ts)
```

Comment:
```suggestion
        elif isinstance(a, Bit):
            ts.extend([a])
```

Reply:
@leonardt I think we can merge this for now. that will be a bigger change and will likely operate on completely separate types. Perhaps the long run is to only use those types, but this is a bit orthogonally useful